### ISSUE TITLE
Question about the number of "out_channels"

### ISSUE THREAD

**Issue by chenhaiwen** [ShihuaHuang95/FaPN-full#3]
After watching your code, i have a question, about every layer after "FSM" and "FAM"  model the output channels are same.Do I understand that right?  And i want to know what the number of  "out_channels" is set to. Thanks a lot.

**Comment by ShihuaHuang95** [comment]
Yes, you are right. And the number of "out_channels" is 256 following the setting of default FPN.
